### DISCUSSION TITLE
Does this work on Version 106.0.5249.112

### DISCUSSION THREAD

**Discussion by ItsLumine** [3kh0/ext-remover#255] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hey. I'm just wondering would this work on this version of chrome os

**Comment by ennzo1010** [comment, marked as answer]
Probably not according to the staff, but they are trying to make a version for 106.0.5249.112 and up.

**Reply by ItsLumine** [reply]
Because my chromebook is at ver 107 right now

**Reply by pl4udible** [reply]
This means it will not work

**Reply by ItsLumine** [reply]
yep. I tried my personal chromebook and it's still at ver 100
